example/index.rst

```diff
@@ -35,7 +35,7 @@ Coverity plugin table
 :latex:`\begin{landscape}`
 
 .. coverity-list:: Coverity defects table and chart
-    :col: CID,Classification,Checker,Comment
+    :col: CID,Classification,Checker,Reference
```

Comment:
add new one, so that we have 2 of these with both examples

Reply:
7ae36da96d056677cf538996919a90af61ed6bc5